src/types/plugin.ts

```diff
@@ -0,0 +1,23 @@
+export interface ConnectOptions {
+
+}
+
+export interface DataHandler {
+  (data: Buffer): Promise<Buffer>
+}
+
+export interface MoneyHandler {
+  (amount: string): Promise<void>
+}
+
+export interface IPlugin {
```

Comment:
Note this is also exported from `ilp-compat-plugin` ([here](https://github.com/interledgerjs/ilp-compat-plugin/blob/master/src/index.ts#L49))

Reply:
Good point, but I think it's worth for the connector to have its own definition. ilp-compat-plugin shouldn't be used for anything other than converting old plugins I think.